### PR TITLE
Enhancement/houdini menu help submenu

### PR DESCRIPTION
**Pull request by MustafaJafar**
Add help links in Houdini menu 
I've added few links: 
- Ayon server
- [Houdini Artist Docs](https://ayon.ynput.io/docs/addon_houdini_artist)
- [Houdini Admin Docs](https://ayon.ynput.io/docs/addon_houdini_admin)
- [Community Forums](https://community.ynput.io/)

image

## Testing notes:
1. Launch Houdini
2. Test links

### CODE REVIEW

File: client/ayon_core/hosts/houdini/startup/MainMenuCommon.xml
Comment (BigRoy) [inline client/ayon_core/hosts/houdini/startup/MainMenuCommon.xml:116]: I've never seen submenus with trailing dots?
```suggestion
            <label>Help</label>
```

The trailing dots usually indicate that clicking it opens something else.

File: client/ayon_core/hosts/houdini/startup/MainMenuCommon.xml
Comment (BigRoy) [inline client/ayon_core/hosts/houdini/startup/MainMenuCommon.xml:129]: Should we label this `AYON Community Forum` instead?
Just so it's clear it's AYON related - not related to whatever studio might have rolled out AYON, etc.

```suggestion
                <label>AYON Community Forum</label>
```

**Review by BigRoy — APPROVED** [review]
Didn't test it - but code looks good to me.

I suppose it's also up for discussion whether adding help entries like these inside the host menus will be good practice going forward. I'll leave that to @mkolar to decide.

**Review by moonyuet — APPROVED** [review]
Looks good....I can access the ayon page and docs through Houdini submenu.
@antirotor can you please take a look to see anything we need to add before the merge? Thanks!

**Comment by mkolar** [comment]
I'm not very happy with merging this to be honest, unless we do it across the board. Our official addons should be defining best practices for other and this is adding more differences between them. 

Useful feature, but I'd like to see it planned and executed on multiple hosts for it to have real impact.

**Comment by MustafaJafar** [comment]
Closing this PR as it doesn't align with other addons.

**Pull request closed without merging**
